mmv1/products/containerattached/Cluster.yaml

```diff
@@ -337,3 +337,26 @@ properties:
         values:
           - :DISABLED
           - :PROJECT_SINGLETON_POLICY_ENFORCE
+  - !ruby/object:Api::Type::NestedObject
+    name: proxyConfig
+    required: false
```

Comment:
Nit: `false` is the default, so this line can be removed

Reply:
Removed, thanks.